docking-station-app/src/components/ActionView/index.tsx

```diff
@@ -0,0 +1,18 @@
+'use client'
+
+import { Container } from '@mantine/core'
+import ActionMenu from './ActionMenu'
+import ActionBar from './ActionBar'
+
+export default function ActionView({ onUpdate, onRefresh }: { onUpdate: () => void, onRefresh: () => void }) {
+  return (
+    <>
+      <Container visibleFrom='sm'>
+        <ActionBar onUpdate={onUpdate} onRefresh={onRefresh} />
+      </Container>
+      <Container hiddenFrom='sm'>
+        <ActionMenu onUpdate={onUpdate} onRefresh={onRefresh} />
+      </Container>
```

Comment:
`Container` is usually a root-level component, it holds the rest of the UI and has some spacing, margin and some other calculations automatically applied to it.

Use `Box` instead.